.github/workflows/push.yml

```diff
@@ -60,6 +60,12 @@ jobs:
           GOOGLE_ANALYTICS: ${{secrets.GOOGLE_ANALYTICS_NON_PROD}}
           NODE_ENV: "dev"
 
+      - name: (NonProd) Deploy
+        if: github.ref == 'refs/heads/master' && github.repository == 'linz/basemaps'
+        run: |
+            cd packages/smoke
+            node --test
```

Comment:
Future Work: Use Lambda URL directly not hitting cloud front? or use cache busting??

Prefer cache busting, so we are testing the entire stack.